src/test/java/Problem2Test.java

```diff
@@ -8,7 +10,41 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
-        // add your tests here
+        int inputsWeSort[][] = {
+                {4,3,2,1},
+                {0},
+                {1,0},
+                {1,4,3,5,7,9,2,100,40,0},
+                {-5,-10,20},
+                {-5,-3,-20,-7},
+                {-5,-3,5,20,-4,-40,500},
+                {-5,-5,3,-5,-5,3},
+                {}
+        };
+
+        int inputsSorted[][] = new int[inputsWeSort.length][];
+        //copy and sort the arrays from inputsWeSort into inputsSorted
+        for (int i = 0; i < inputsWeSort.length; i++) {
+            inputsSorted[i] = Arrays.copyOf(inputsWeSort[i],inputsWeSort[i].length);
+            Arrays.sort(inputsSorted[i]);
+        }
+        //check if the new array is the same length as the old one
+        assertEquals(inputsWeSort.length, inputsSorted.length);
+
+        for (int i=0; i<inputsSorted.length; i++) {
+            Problem2.bubbleSort(inputsWeSort[i]);
+            //https://www.java2novice.com/junit-examples/assert-array-equals/
+            assertArrayEquals(inputsWeSort[i], inputsSorted[i]);
```

Comment:
the order of argument is reversed. first argument should be the expected value, and 2nd argument is the actual value.